includes/functions.php

```diff
@@ -159,12 +161,19 @@ function _h($level, $html){
     $hid = $base_hid.'-'.$i;
     $i += 1;
   }
+  # Class for hiding in ToC
+  $toc_hidden_class = $toc_hidden ? 'toc-hidden' : '';
   return '
-    <h'.$level.' id="'.$hid.'">
+    <h'.$level.' id="'.$hid.'" class="'.$toc_hidden_class.'">
       <a href="#'.$hid.'" class="header-link"><span class="fas fa-link"></span></a>
       '.$html.'
     </h'.$level.'>';
 };
+function _h1($html){ return _h(1, $html); }
```

Comment:
why do we need this nested functions? I think they just add an unneeded complexity to the code because `_h()` is already used in different parts, so one is familiar with it and `_h1()` is not much less to write than `_h(1,)`...

Reply:
We don't need it, I just thought that `_h1` was nicer to write - it's more obvious what it does when you're reading the PHP code.

It's only used in one place so far but I was thinking of trying to use them more through the rest of the code base as I think that the anchors are written manually in a bunch of places.